### PR TITLE
Fix typo in wiki of KvStore

### PR DESCRIPTION
**Pull request by cfdream**
# Pull Request Template

Title: [<module-name>] <Breif title under 80 characters>

Description:

- Explain in detail what was broken!
- How did you fix it

Test Plan:

- Explain in detail how you tested it

# Pull Request Guideline

- Before sending a pull request make sure each commit solves one clear, minimal,
  plausible problem. Further each commit should have the following format:
- Please avoid sending a pull request with recursive merge nodes, as they
  are impossible to fix once merged. Please rebase your branch on
  facebook/openr master instead of merging it.
- If you are a new contributor please have a look at our contributing guidelines:
  [CONTRIBUTING](https://github.com/facebook/openr/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: openr/docs/Protocol_Guide/KvStore.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ getKvStoreKeyVals(std::string area, thrift::KeyGetParams keyGetParams)
  * @return: None
  */
 void
-setKvStoreKeyVals(std::string area, thrift::KeyGetParams keyGetParams)
+setKvStoreKeyVals(std::string area, thrift::KeySetParams keySetParams)
```
Comment (cooperlees) [inline openr/docs/Protocol_Guide/KvStore.md:83]:
```suggestion
setKvStoreKeyVals(std::string area, thrift::KeySetParams KeySetParams)
```

Is this a typo in the typo fix?

**Review by cooperlees — COMMENTED** [review]
Can @xiangxu1121 double check please :)

File: openr/docs/Protocol_Guide/KvStore.md (hunk 1 above)
Comment (cfdream) [reply openr/docs/Protocol_Guide/KvStore.md:83]: For parameter names, we are following camel naming format, and first character is lower case? Thus, the fix does not have typo?

Comment (cooperlees) [reply openr/docs/Protocol_Guide/KvStore.md:83]: Sorry, yeah my bad, I thought this was two types.

**Pull request closed without merging**